src/test/java/teammates/sqllogic/core/FeedbackQuestionsLogicTest.java

```diff
@@ -29,6 +29,7 @@ public class FeedbackQuestionsLogicTest extends BaseTestCase {
     private FeedbackQuestionsDb fqDb;
     private UsersLogic usersLogic;
     // private FeedbackResponsesLogic frLogic;
+    // private FeedbackSessionsLogic feedbackSessionsLogic;
```

Comment:
Any reason why we are adding a comment here and initialising the logic in setUpMethod anyway? Understand that the same is being done for feedback responses logic, but standardising initialisation now can help save some time down the line.

Reply:
Ah! These have been initialised in the scope of the setUpMethod but weren't removed. Let me remove them, also from feedback responses logic.

Reason why we commented out was checkstyle/lint failed so we had to localise the initialisation in the scope of the set up method itself, applies to all other classes too but yes it's better to remove the commented code and change the scope of the mock variables when we need them in the future 👍🏻

Reply:
Thanks for the clarification! By "same is done for feedback responses logic" I meant how line 31 was also commented out haha.